collectors/auth0/auth0_collector.js

```diff
@@ -27,15 +27,17 @@ const tsPaths = [
 
 const HOSTNAME_REGEXP = /^[htps]*:\/\/|\/$/gi;
 
+const STREAM_NAME = 'default';
```

Comment:
we need to extract this constant into the library because it is used in multiple collectors.

Reply:
I have removed default value for stream and it will post the status with collector type  (ex paws_okta_status) as it handle d in Paws and al-aws-collector while [post error status](https://github.com/alertlogic/al-aws-collector-js/pull/62/files#diff-c06f5f0e08865e628912333b6bb7941cf34fcc02b6fc4163a80e350b5bfd4c47R188) and [post ok status](https://github.com/alertlogic/al-aws-collector-js/pull/62/files#diff-c06f5f0e08865e628912333b6bb7941cf34fcc02b6fc4163a80e350b5bfd4c47R420)